tradingstrategy/binance/candlestick.py

```diff
@@ -0,0 +1,221 @@
+"""Get candlestick price and volume data from Binance.
+"""
+
+import requests
+import datetime
+import pandas as pd
+import numpy as np
+import logging
+
+from tradingstrategy.timebucket import TimeBucket
+from pathlib import Path
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_binance_candlestick_data(
```

Comment:
Rename this to `fetch_` so it is aligned with other function names in this package.

- get: read from memory
- fetch: read over HTTP